### PR TITLE
[no-jira] Change to flow resource docs

### PR DESCRIPTION
**Pull request by charliecon**

### CODE REVIEW

File: genesyscloud/resource_genesyscloud_flow.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ func FlowExporter() *resourceExporter.ResourceExporter {
 
 func ResourceFlow() *schema.Resource {
 	return &schema.Resource{
-		Description: `Genesys Cloud Flow`,
+		Description: `Genesys Cloud Architect Flow. Note: Changing any field in this resource will cause the current flow to be dropped and recreated with a new ID.`,
```
Comment (dginty4) [inline genesyscloud/resource_genesyscloud_flow.go:61]: Is the old flow deleted? If not is that worth mentioning?

Comment (charliecon) [reply genesyscloud/resource_genesyscloud_flow.go:61]: @dginty4 Yeah it is deleted. That's what "dropped" is supposed to mean in this case

Comment (dginty4) [reply genesyscloud/resource_genesyscloud_flow.go:61]: Ah ok, sorry I thought it was said in standup it wasn't deleted, my bad

**Review by dginty4 — APPROVED** [review]

File: genesyscloud/resource_genesyscloud_flow.go (hunk 1 above)
Comment (charliecon) [reply genesyscloud/resource_genesyscloud_flow.go:61]: Oh yeah that bug was fixed in another PR that's been merged already

**Pull request closed without merging**